builtin/mainmenu/modmgr.lua

```diff
@@ -294,16 +294,18 @@ function modmgr.get_dependencies(modfolder)
 		local soft_dependencies = {}
 		local dependencyfile = io.open(filename,"r")
 		if dependencyfile then
-			local dependency = dependencyfile:read("*l")
-			while dependency do
+			local contents = dependencyfile:read('*a')
```

Comment:
I think so. If it doesn't need to handle CR, just stripping CRs from CRLFs should be trivial. Will reduce the change.